vignettes/tutorial.Rmd

```diff
@@ -227,6 +227,23 @@ download_url <- result$sensitive_unshuffled_avatars_datasets$download_url
 sensitive_unshuffled_avatars <- get_dataset(download_url, columns)
 ```
 
+### Setting up an email account
+
+
+*This section is only needed if the use of emails to login is
+activated in the global configuration. It is not the case by default.*
+
+At this moment, you have to get in touch with your Octopize contact so that they can
```

Comment:
```suggestion
At the moment, you have to get in touch with your Octopize contact so that they can
```
Not sure if it's typo or mistake, but this does not mean what you want